routing/Handlers.php

```diff
@@ -18,6 +18,11 @@ function other($otherFile){
             require __DIR__.'/../LEAF_Request_Portal/'.$otherFile;
         }
 
+        function favicon($favicon){
+            header('Content-Type: image/png');
+            readfile(__DIR__.'/../LEAF_Request_Portal/'.$favicon);
```

Comment:
Codacy is going to complain about this later for spacing in between ```.```

Reply:
addressed